src/LiveComponent/assets/src/live_controller.ts

```
@@ -179,6 +182,14 @@ export default class extends Controller {
 
                         break;
                     }
+                    case 'files':
+                        this.fileTargets.forEach(input => {
+                            if (!modifier.value || input.name === modifier.value) {
+                                files[input.name] = input.files;
+                            }
+                        })
```

Could we throw an exception if `modifier.value` IS passed but we couldn't find any of those? Maybe something like:

> Could not find the file input `foo`. Did you remember to make this element a Stimulus target (e.g. `{{ stimulus_target('live', 'file') }}`).

Yes, I know it's weird to put Twig code into the JS message, but it will be the most helpful to the most people.

---

I've done as suggested :+1: